dockstore-common/src/main/java/io/dockstore/common/EntryUpdateTime.java

```diff
@@ -0,0 +1,40 @@
+package io.dockstore.common;
+
+import java.sql.Timestamp;
+import java.util.Date;
+
+public class EntryUpdateTime {
```

Comment:
Ditto, if it shows up in the openapi.yaml , it needs OpenAPI annotations (or less desired, swagger annotations)